src/core/actions/search.js

```diff
@@ -5,10 +5,10 @@ import {
 } from 'core/constants';
 
 
-export function searchStart({ filters, page, results }) {
+export function searchStart({ errorHandlerId, filters, page, results }) {
```

Comment:
Fair enough. I think results might not actually be required in a refactor later but I will make them required for now.